Setup/template integration

atariq12382 opened the pull request.

Sultan0099 commented on app/helpers/managers_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module ManagersHelper
```
Remove this file if it is not in use.

Sultan0099 commented on app/javascript/packs/application.js:
```diff
@@ -13,3 +13,72 @@ import "bootstrap"
 Turbolinks.start()
 ActiveStorage.start()
+// import "../../../vendor/assets/js/apexcharts.custom"
+// import "../../../vendor/assets/js/apexcharts.min"
```
Remove commented code and why did you use require instead of import?

atariq12382 commented:
@Sultan0099 both of the issues has been resolved. I have removed managers_helper.rb file and omitted all the commented code and replaced the required with the import
